README.md

```diff
@@ -26,4 +28,26 @@ lessons, which are available under the
 
 ## Setup
 
+### Installing Python Using Anaconda
```

Comment:
I would not think that the beginners would be able to judge if to install Anaconda or Miniconda and why they are given two choices. I would suggest just leaving the miniconda instructions

Reply:
I agree with @maikia. I would keep a single set of instructions, namely those using Miniconda.